src/app/api/payments/pawapay/routes.ts

```
@@ -0,0 +1,279 @@
+      case 'payoutStatus': {
+        const payoutId = searchParams.get('payoutId');
+        if (!payoutId) {
+          return NextResponse.json({ error: 'payoutId parameter is required' }, { status: 400 });
+        }
+        const status = await pawaPay.checkPayoutStatus(payoutId);
+        return NextResponse.json(status);
+      }
+      
+      // Default to availability check
+      default:
+        const availability = await pawaPay.checkAvailability(country);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)